src/main.cpp

```diff
@@ -40,44 +41,58 @@
 #include "SeisSol.h"
 #include "Initializer/InitProcedure/Init.hpp"
 
-#include <yaml-cpp/yaml.h>
+#ifdef ACL_DEVICE
+#include "device.h"
+#endif
+
+int main(int argc, char* argv[]) {
+#if defined(ACL_DEVICE) && defined(USE_MPI)
+  MPI::mpi.bindAcceleratorDevice();
+#endif
 
-int main(int argc, char* argv[])
-{
-        LIKWID_MARKER_INIT;
+#ifdef ACL_DEVICE
```

Comment:
Merge with the `#if` from above? I.e.

```
#ifdef ACL_DEVICE
#ifdef USE_MPI
MPI::mpi.bindAcceleratorDevice();
#endif
  device::DeviceInstance& device = device::DeviceInstance::getInstance();
  device.api->initialize();
  device.api->allocateStackMem();
#endif
```
or so?

Reply:
done